controllers/jobs.js

```diff
@@ -0,0 +1,77 @@
+const Tires = require('../models/Job')
+const { StatusCodes } = require('http-status-codes')
+const { BadRequestError, NotFoundError } = require('../errors')
+
+
+const getAllJobs = async (req, res) => {
+    const tires = await Tires.find({ createdBy: req.user.userId }).sort('createdAt')
```

Comment:
Love that you’re already thinking of final project! If you want to update the routers to be tire rather than job for readability feel free in the next assignment